.github/workflows/main.yml

```diff
@@ -77,3 +78,75 @@ jobs:
           npm run test-install
           # Undo those changes.
           git checkout --force
+
+  macos:
+    # https://github.com/actions/virtual-environments#available-environments
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+          ref: ${{ github.head_ref }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          # Test only the oldest maintenance LTS Node.js version.
+          # https://github.com/nodejs/Release#release-schedule
+          node-version: 10
+
+      - name: Install dependencies
+        run: |
+          # Test platform-specific browser binary fetching for both
+          # Chromium and Firefox.
+          PUPPETEER_PRODUCT=firefox npm install
+          npm install
+          ls .local-chromium .local-firefox
```

Comment:
Is it for debugging reasons? Do we need it?

Reply:
Our previous Travis CI setup did the same thing. I left it in on purpose, as it improves the log output (which helps to post-mortem-debug CI failures).